### PR TITLE
[ci]: Merge azure pipelines from master to 202012 branch

### PR DESCRIPTION
**Pull request by xumia**
**What I did**
Merge azure pipelines from master to 202012 branch

**Why I did it**

**How I verified it**

**Details if related**

### CODE REVIEW

File: .azure-pipelines/build-docker-sonic-vs-template.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  steps:
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      source: specific
+      project: build
+      pipeline: 9
+      artifact: ${{ parameters.swss_common_artifact_name }}
+      runVersion: 'latestFromBranch'
+      runBranch: 'refs/heads/master'
```
Comment (qiluo-msft) [inline .azure-pipelines/build-docker-sonic-vs-template.yml:42]: Why master branch?

Comment (xumia) [reply .azure-pipelines/build-docker-sonic-vs-template.yml:42]: I need to fix it, because there were no 202012 artifacts when sending the PR.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**